magicctapipe/scripts/lst1_magic/lst1_magic_mc_dl0_to_dl1.py

```diff
@@ -250,7 +250,11 @@ def mc_dl0_to_dl1(input_file, output_dir, config, muons_analysis):
             n_events_skipped = 0
             n_events_processed = 0
 
-            event_source_allowed_tels = EventSource(input_file, allowed_tels=list(mc_tel_ids.values()))
+            event_source_allowed_tels = EventSource(
+                input_file,
+                allowed_tels=list(mc_tel_ids.values()),
+                focal_length_choice="effective"
```

Comment:
there is another event_source in line 127
that one is used to get the subarray and also to save the simulation config. I'm not 100% sure that it is needed, but maybe safer to also read effective focal length there?

Reply:
Hi Julian, indeed you are right. Actually this is critical, because that EventSource is used to get the focal length from the subarray. I commited a change for this.